projects/ad7134_fmc/common/ad7134_bd.tcl

```diff
@@ -81,6 +97,29 @@ ad_ip_parameter axi_ad7134_dma CONFIG.DMA_2D_TRANSFER 0
 ad_ip_parameter axi_ad7134_dma CONFIG.DMA_DATA_WIDTH_SRC [expr $data_width * $adc_num_of_channels]
 ad_ip_parameter axi_ad7134_dma CONFIG.DMA_DATA_WIDTH_DEST 128
 
+# odr generator
+
+ad_ip_instance axi_pwm_gen odr_generator
```

Comment:
The odr_generator doesn't seem to need async_clk_en, given that it works on the 100MHz/system clock